src/tests/functional/plugin/cpu/shared_tests_instances/skip_tests_config.cpp

```diff
@@ -175,6 +175,9 @@ std::vector<std::string> disabledTestPatterns() {
         // Issue: 75022
         R"(.*OVExecutableNetworkBaseTest.*LoadNetworkToDefaultDeviceNoThrow.*)",
         R"(.*IEClassBasicTest.*LoadNetworkToDefaultDeviceNoThrow.*)",
+        // Issue: 76369
+        // Not supported Gather operation version. CPU plug-in supports only 7 version.
```

Comment:
Does Gather really affect this?

Reply:
yes, because it is used into `Optimize*SequenceTransposes`
https://github.com/openvinotoolkit/openvino/blob/master/src/plugins/intel_cpu/src/ngraph_transformations/rnn_sequences_optimization.cpp#L59